### DISCUSSION TITLE
How do you disable all the debug output produced by the OpenTelemetry sdk?

### DISCUSSION THREAD

**Discussion by jakenuts** [open-telemetry/opentelemetry-dotnet#4775] [Q&A]
For the life of me I cannot figure out how to silence all the events being displayed in my output window. Tried changing Logging configuration a hundred ways, nothing seems to work.

Thanks!

**Comment by cijothomas** [comment]
Can you share a screenshot of what you are seeing? its not very obvious what is the debug output you are referring to.
Likely this : https://github.com/open-telemetry/opentelemetry-dotnet/blob/main/src/OpenTelemetry/Internal/OpenTelemetrySdkEventSource.cs#L33-L35 ?
